packages/integrations/sitemap/README.md

```diff
@@ -1,115 +1,160 @@
+
 Now, [build your site for production](https://docs.astro.build/en/reference/cli-reference/#astro-build) via the `astro build` command. You should find your sitemap under `dist/sitemap.xml`!
 
-You can also check our [Astro Integration Documentation][astro-integration] for more on integrations.
+> **Warning**
+> If you forget to add a `site`, you'll get a friendly warning when you build, and the `sitemap.xml` file won't be generated.
 
 ## Configuration
 
-### filter
+To configure this integration, pass an object to the `sitemap()` function call in `astro.config.mjs`.
 
-All pages are included in your sitemap by default. By adding a custom `filter`, you can filter included pages by URL.
+__astro.config.mjs__
+```js
+...
+export default defineConfig({
+  integrations: [sitemap({
+    filter: ...
+  })]
+});
+```
+
+<details>
+  <summary><strong>filter</strong></summary>
+  
+  <br/>
+  
+  All pages are included in your sitemap by default. By adding a custom `filter` function, you can filter  included pages by URL.
 
 __astro.config.mjs__
 
 ```js
-import sitemap from '@astrojs/sitemap';
-
-export default {
-  site: 'https://stargazers.club',
-  integrations: [
+...
     sitemap({
       filter: (page) => page !== 'https://stargazers.club/secret-vip-lounge'
     }),
-  ],
-}
 ```
 
-The `page` function parameter is the full URL of your rendered page, including your `site` domain. Return `true` to include a page in your sitemap, and `false` to remove it.
+The function will be called for every page on your site. The `page` function parameter is the full URL of the page currently under considering, including your `site` domain. Return `true` to include the page in your sitemap, and `false` to leave it out.
+  
+</details>
 
-### canonicalURL
-
-If present, we use the `site` config option as the base for all sitemap URLs. Use `canonicalURL` to override this.
+<details>
+  <summary><strong>customPages</strong></summary>
+  
+  <br/>
+  
+  In some cases, a page might be part of your deployed site but not part of your Astro project.
+  If you'd like to include a page in your sitemap that _isn't_ created by Astro, you can use this option.
 
 __astro.config.mjs__
 
 ```js
-import sitemap from '@astrojs/sitemap';
-
-export default {
-  site: 'https://stargazers.club',
-  integrations: [
+...
     sitemap({
-      // https://astronaut.party will be used for all sitemap URLs instead
-      canonicalURL: 'https://astronaut.party',
+      customPages: ['https://stargazers.club/external-page', 'https://stargazers.club/external-page2']
     }),
-  ],
-}
 ```
+</details>
+
+## Examples
+- The official Astro website uses Astro Sitemap to generate [its sitemap](https://astro.build/sitemap.xml).
+- The [https://github.com/withastro/astro/tree/latest/examples/integrations-playground?on=github](integrations playground template) comes with Astro Sitemap installed. Try adding a route and building the project!
```

Comment:
Classic backwards Markdown link syntax. Not sure how the syntax authors ever expected people to remember this.

```suggestion
- The [integrations playground template](https://github.com/withastro/astro/tree/latest/examples/integrations-playground?on=github) comes with Astro Sitemap installed. Try adding a route and building the project!
```